main/res/raw-fi/changelog_base.md

```diff
@@ -15,9 +15,10 @@
 
 ### Muu
 - Uusi: Pikalataa geokoodit leikepöydällä olevasta tekstistä päänäytön hakukenttään
-- Uusi: Tee asetukset => Näytä asetukset suodatettavaksi
-- Uusi: Lisätty GC Wizard hyödyllisten sovellusten luetteloon
-- Muutos: Poistettiin viivakoodiskanneri hyödyllisten sovellusten listalta ja päävalikosta
-- Muutos: Poistettiin BRouter hyödyllisten sovellusten luettelosta (voit silti käyttää sekä ulkoista että sisäistä navigointia)
-- Korjaus: Vältä toistuvia päivitysten karttojen/reitityslaattojen tarkistuksia välillä = 0
-- Korjaus: Optimoi tuki salasanojen automaattiseen täyttämiseen ulkoisesta salasanasovelluksista asetuksissa
```

Comment:
@moving-bits and others:
Just a quick note: It seems we should avoider reordering the existing lines in the changelogs. Seemingly this will invalidate all existing translations (still in the Crowdin memory, but translators need to reapprove them).